make.ps1

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
I think it would be better if stick to one entrypoint, the `Makefile` instead of introducing a script here. Maybe you don't have make installed? I think the makefile would work as-is.

Reply:
I've tried using the `make` command on the makefile under cygwin with success, but I'm worried that GitHub actions may not support this software. I'll look into it (if you want to) so that automatic testing can also work under windows.